diktat-rules/src/test/kotlin/org/cqfn/diktat/ruleset/chapter3/StringTemplateRuleWarnTest.kt

```diff
@@ -35,10 +35,12 @@ class StringTemplateRuleWarnTest : LintTestBase(::StringTemplateFormatRule) {
                     |class Some { 
                     |   val template = "${'$'}{a} ${'$'}{asd.moo()}"
                     |   val some = "${'$'}{1.0}"
+                    |   val another = "${'$'}{1}"
                     |}
                 """.trimMargin(),
                 LintError(2, 20, ruleId, "${Warnings.STRING_TEMPLATE_CURLY_BRACES.warnText()} ${'$'}{a}", true),
-                LintError(3, 16, ruleId, "${Warnings.STRING_TEMPLATE_CURLY_BRACES.warnText()} ${'$'}{1.0}", true)
+                LintError(3, 16, ruleId, "${Warnings.STRING_TEMPLATE_CURLY_BRACES.warnText()} ${'$'}{1.0}", true),
+                LintError(4, 19, ruleId, "${Warnings.STRING_TEMPLATE_CURLY_BRACES.warnText()} ${'$'}{1}", true)
```

Comment:
Do you mean that braces can be omitted around dollar sign? Then you must ensure that your fixer escapes it for single-quoted strings and doesn't fix `${'$'}` for triple-quoted strings (no way to escape `$` there)

Reply:
No, it was meant that braces can be omitted around integers or floats

Reply:
But they can't - in your example it's not a part of template, these braces are just regular characters inside the string.

Reply:
Oh, this warning complains about `$`, not `1`? Then it makes sense.. Do you have tests that `${'$'}` is considered valid in triple-quoted strings?